contrib/win32/openssh/appveyor.psm1

```diff
@@ -494,43 +500,15 @@ function Deploy-OpenSSHTests
     }
     
 
-    [System.IO.DirectoryInfo] $repositoryRoot = Get-RepositoryRoot    
+    [System.IO.DirectoryInfo] $repositoryRoot = Get-RepositoryRoot
     
     $sourceDir = Join-Path $repositoryRoot.FullName -ChildPath "regress\pesterTests"
-    Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Include *.ps1,*.psm1 -Force -ErrorAction Stop
+    Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Include *.ps1,*.psm1, sshd_config -Force -ErrorAction Stop
 
-    $sourceDir = Join-Path $repositoryRoot.FullName -ChildPath "bin\$folderName\$RealConfiguration"    
+    $sourceDir = Join-Path $repositoryRoot.FullName -ChildPath "bin\$folderName\$RealConfiguration"
     Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Exclude ssh-agent.exe, sshd.exe -Force -ErrorAction Stop
 
-
-    $sshdConfigFile = "$OpenSSHTestDir\sshd_config"
-    if (-not (Test-Path -Path $sshdConfigFile -PathType Leaf))
-    {
-        Write-BuildMessage "Installation dependencies: $OpenSSHTestDir\sshd_config is missing in the folder" -Category Error
-        throw "$OpenSSHTestDir\sshd_config is missing in the folder"
-    }
-
-    if ($env:DebugMode)
-    {
-        $strToReplace = "#LogLevel INFO"
-        (Get-Content $sshdConfigFile).Replace($strToReplace,"LogLevel Debug3") | Set-Content $sshdConfigFile
-    }
-    if(-not ($env:psPath))
-    {
-        $psCorePath = GetLocalPSCorePath
-        Set-BuildVariable -Name psPath -Value $psCorePath
-    }    
-
-    $strToReplace = "Subsystem	sftp	sftp-server.exe"
-    if($env:psPath)
-    {
-        $strNewsubSystem = @"
-Subsystem	sftp	sftp-server.exe
-Subsystem	powershell	$env:psPath
-"@
-    }
-
-    (Get-Content $sshdConfigFile).Replace($strToReplace, $strNewsubSystem) | Set-Content $sshdConfigFile
+    Restart-Service sshd
```

Comment:
This is test deployment. Before test deployment, the product binaries already got installed and service got started.